src/errors/not-an-article.ts

```diff
@@ -0,0 +1,6 @@
+export default class NotAnArticle extends Error {
+  constructor(type?: string | Array<string>) {
```

Comment:
I’d avoid doing ‘thing or list of thing’ as it’s awkward to handle (see next line). Better to always have an array argument, or use a varadic (`...`).